test/gtest/ucs/test_event_set.cc

```diff
@@ -34,8 +34,7 @@ class test_event_set : public ucs::test {
 
 const char *test_event_set::evfd_data = UCS_EVENT_SET_TEST_STRING;
 
-static void event_set_func1(int fd,
-                                    int events)
+static void event_set_func1(int fd, int events, ucs_event_set_extra_args **args)
```

Comment:
TODO: `ucs_event_set_extra_args **args` assetion test. Maybe I need to create another PR for this change.(extent event_set API)

Reply:
Yes, it would be good to have UCS event set API changes in the separate PR (+ tests that covers this functionality)